docs/guides/getting-started/prerequisites.md

```diff
@@ -72,6 +72,8 @@ successful, the following line will appear:
 Rust is installed now. Great!
 ```
 
+Make sure to restart your Terminal for the changes to take effect.
```

Comment:
```suggestion
Make sure to restart your terminal for the changes to take effect.
```

Reply:
i'm too german for this stuff

Reply:
😂 I love the random capitalisations scattered throughout the docs